src/app/routes/api/v3/forms/public-forms.submissions.routes.ts

```diff
@@ -65,3 +65,10 @@ PublicFormsSubmissionsRouter.route(
   limitRate({ max: rateLimitConfig.submissions }),
   EncryptSubmissionController.handleStorageSubmission,
 )
+
+PublicFormsSubmissionsRouter.route(
+  '/:formId([a-fA-F0-9]{24})/submissions/storage/presigned-post-data',
```

Comment:
suggestion: maybe `submissions/storage/get-s3-presigned-post-data` would be more descriptive? 😆

Reply:
Okay! Done in [1fcb5e3](https://github.com/opengovsg/FormSG/pull/6685/commits/1fcb5e38dde976d92389f90d14b66c943569075d).